sources/mysql/mysqldump.go

```diff
@@ -23,13 +23,16 @@ import (
 
 	"github.com/cloudspannerecosystem/harbourbridge/internal"
 	"github.com/cloudspannerecosystem/harbourbridge/schema"
+	"github.com/cloudspannerecosystem/harbourbridge/sources/common"
 	"github.com/pingcap/parser"
 	"github.com/pingcap/parser/ast"
 	"github.com/pingcap/parser/opcode"
 	"github.com/pingcap/tidb/types"
 	driver "github.com/pingcap/tidb/types/parser_driver"
 )
 
+type DbDumpImpl struct{}
+
```

Comment:
nit; type definition should come after var declarations and preferably right before `func (ddi DbDumpImpl) GetToDdl()` etc. (similarly in other files)